features/steps/step_selenium_alias_hipchat.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: UTF-8 -*-
+
+# @mark.steps
+# ----------------------------------------------------------------------------
+# STEPS:
+# ----------------------------------------------------------------------------
+from behave import when, then
+
+
+@when('we create new alias in chat')
+def step_impl(context):
+
+    context.lobby_page.navigate()
+    context.lobby_page.random_click()
+    context.lobby_page.open_alias_room()
+    assert context.lobby_page.chat_adding_alias()
+
+@when('we create new alias with click buttons')
+def step_impl(context):
+    context.lobby_page.delete_alias_in_chat_comands()
+    context.lobby_page.open_alias_menu()
+    context.lobby_page.open_menu()
+    context.lobby_page.focus_at_frame()
```

Comment:
Ideally, test steps should not be aware of implementation detail. "Frame" and "iframe" are quite specific to web.